tests/test_config.py

```diff
@@ -7,20 +7,25 @@
 PATH = os.path.dirname(os.path.abspath(__file__))
 test_creds = os.path.join(PATH, "test_creds.yml")
 
+
 def test_preconditions():
     assert os.path.exists(test_creds)
 
-@mock.patch.dict(os.environ, {"N26_USER": "john.doe@example.com", "N26_PASSWORD": "$upersecret", "N26_CARD_ID": "123456789"})
+
+@mock.patch.dict(os.environ,
+                 {"N26_USER": "john.doe@example.com", "N26_PASSWORD": "$upersecret", "N26_CARD_ID": "123456789"})
```

Comment:
This part looks a little off. I would suggest to roll it back and I can take care about better formatting.

Reply:
I have removed the test files form this commit.
Please have a look :eyes: